src/routes/auth.route.js

```diff
@@ -0,0 +1,39 @@
+import { Router } from 'express';
+import { authController } from './../controllers/index.js';
+import { authMiddleware } from './../middlewares/index.js';
+import { catchError } from './../utils/catchError.js';
+
+export const authRouter = new Router();
+
+authRouter.post('/registration', catchError(authController.register));
```

Comment:
Ensure that the `register` function in `authController` is correctly implemented and handles registration logic as expected.